### ISSUE TITLE
Multisync issue when applying effects to a remote model

### ISSUE THREAD

**Issue by cbennett6008** [adamcoulombe/fpp-WledEffects#2]
I have the WLED plugin installed on my main player.  
Main Player:  a RasPi 3b+
    Channel Outputs: 8 DPP based outputs with only one currently active (it's July. :) )
    Pixel Overlay models are created on the DPP controlers mentioned above, There are no truly local strings.

Remote Player: a BBB with a K8-Scroller cape.
   This player has a 3x2 P5 Panel connected to it.

When I first come into the plugin, without multisync enabled, none of the Pixel Overlay models on the remote DPP controllers appear in the list.  After checking multisync, the DPP based Pixel Overlays appear in the list. 
If I leave All Models selected and apply an effect. nothing happens.  If I uncheck all models, and check the only model that's actually plugged in right now, (my permanent eves lights) and apply an effect, it works as intended.
If I go back and check every pixel overlay that is 'local-ish' to this controller, and apply an effect, we're still working as we intend to.

The problem appears to be if I select any of the Overlay Models that are connected to the remote fpp system.  When one of those is checked, the bigbuttons stop performing any action.  

Please let me know what I information I can provide that would be helpful.

Thanks,
Chris

**Comment by adamcoulombe** [comment]
just to clarify, is the remote system powered on?

again if you could provide your console log  upon reproducing the behavior that may help give some hints as to what the issue is.

**Comment by Crookshank314** [comment]
My WLED plug-in was just downloaded 4 days ago and worked great with all UI working properly until tonight. Now I show patterns but no models and no colors.

Just curious if anyone had found a solution. I took an update that was just released and did a restart with no luck. Deleted the plug-in and re-downloaded it with no luck.

Thanks in advance!

Brad

**Comment by adamcoulombe** [comment]
> My WLED plug-in was just downloaded 4 days ago and worked great with all UI working properly until tonight. Now I show patterns but no models and no colors.
> 
> Just curious if anyone had found a solution. I took an update that was just released and did a restart with no luck. Deleted the plug-in and re-downloaded it with no luck.
> 
> Thanks in advance!
> 
> Brad

Hi Brad. If you would be able to post the output of your browser’s developer console(F12 on your keyboard, then find the console tab), that should give me an idea where to start looking

**Comment by Crookshank314** [comment]
Hi Adam, 

Sorry for the delay. I kept trying to fix the. Issue after sending the question and I crashed my FPP. Of course I did not have a backup even though they tell you to back up. I am currently trying to get back up and running but can't get my BBB to connect to my network properly. For some reason I can't log in to the system through the IP address I assign. I have put a new SD card in but keep hitting roadblocks. 

If I get it back and running and can't get the WLED effects to work I will send you the logs, Thanks!

Brad
